src/write_sensu.c

```diff
@@ -39,6 +39,53 @@
 #include <stddef.h>
 
 #include <stdlib.h>
+#ifdef HAVE_KERNEL_SOLARIS
```

Comment:
@rubenk Yep, that's perfect, thanks! Looks good to me, but it's probably better to listen to @dago with regards to the actual issue at hand ;)